snomed/com.b2international.snowowl.snomed.importer.rf2/src/com/b2international/snowowl/snomed/importer/rf2/util/SnomedRefSetNameCollector.java

```diff
@@ -67,210 +73,219 @@
  */
 public class SnomedRefSetNameCollector {
 	
-	private static final int LARGE_WORK_REMAINING = 10000;
-
-	private static final Map<String, String> URL_TO_ICON_TYPE_ID_MAP = ImmutableMap.<String, String>builder().
-			put(ErroneousAustralianReleaseFileNames.ERRONEOUS_AU_20120229_STRENGTH_REFSET_NAME, Concepts.REFSET_CONCRETE_DOMAIN_TYPE_AU).
-			put(ErroneousAustralianReleaseFileNames.ERRONEOUS_AU_20120229_SUBPACK_QUANTITY_FULL_REFSET_NAME, Concepts.REFSET_CONCRETE_DOMAIN_TYPE_AU).
-			put(ErroneousAustralianReleaseFileNames.ERRONEOUS_AU_20120229_UNIT_OF_USE_QUANTITY_REFSET_NAME, Concepts.REFSET_CONCRETE_DOMAIN_TYPE_AU).
-			put(ErroneousAustralianReleaseFileNames.ERRONEOUS_AU_20120229_UNIT_OF_USE_SIZE_REFSET_NAME, Concepts.REFSET_CONCRETE_DOMAIN_TYPE_AU).
-			put(ErroneousAustralianReleaseFileNames.ERRONEOUS_AU_20110531_CTV3_ID_REFSET_NAME, Concepts.REFSET_SIMPLE_MAP_TYPE).
-			put(ErroneousAustralianReleaseFileNames.ERRONEOUS_AU_20110531_SNOMED_RT_REFSET_NAME, Concepts.REFSET_SIMPLE_MAP_TYPE).
-			put(ErroneousAustralianReleaseFileNames.ERRONEOUS_AU_20110531_SNOMED_LANGUAGE_REFSET_NAME, Concepts.REFSET_LANGUAGE_TYPE).
-			put(ErroneousAustralianReleaseFileNames.ERRONEOUS_AU_20120229_SNOMED_LANGUAGE_REFSET_NAME, Concepts.REFSET_LANGUAGE_TYPE).build();
+	private static final String UNLABELED_REFSET_SUFFIX = " (unresolved)";
+	private static final String ESCAPED_UNLABELED_REFSET_SUFFIX = CharMatcher.is(')')
```

Comment:
`Pattern#quote(String)` will return a String that is treated as a "literal pattern" in regular expressions; suggesting to use that instead of targeted replacements.